src/Pages/Overview/App.js

```diff
@@ -0,0 +1,217 @@
+    const search = this.state.search.trim().toLowerCase()
+    return search !== null || search !== ''
+      ? this.state.users.filter(data =>
+        data.firstName.toLowerCase().includes(search.toLowerCase())
+      )
+      : this.state.users
+  }
+
+  render () {
+    return (
+      <div className='layout'>
+        <h1>Users Dashboard</h1>
+
+        <h6>Search by first-name</h6>
+        <input
+          onChange={e => {
+            this.updateSearch(e)
+          }}
+        />
+
+        <table className='content-table' id='users'>
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Door Code</th>
+              <th>Printing</th>
+              <th>Email Verified</th>
+              <th>Type</th>
+              <th />
+              <th />
+            </tr>
+          </thead>
+
+          <tbody
+            onClick={() => {
+              this.forceUpdate()
+            }}
+          >
+            {this.search(this.state.users).map((user, index) =>
+              display.displayBoard(user, index)
+            )}
+          </tbody>
+        </table>
+
+        <Modal isOpen={display.getToggle()}>
+          <svg
+            width='24'
+            height='24'
+            viewBox='0 0 24 24'
+            onClick={() => {
+              this.callDatabase()
+              display.toggler()
+              this.forceUpdate()
+            }}
+            style={{
+              position: 'relative',
+              marginTop: '5px',
+              marginLeft: '-5px',
+              left: '95%'
+            }}
+          >
+            <path d='M24 20.188l-8.315-8.209 8.2-8.282-3.697-3.697-8.212 8.318-8.31-8.203-3.666 3.666 8.321 8.24-8.206 8.313 3.666 3.666 8.237-8.318 8.285 8.203z' />
+          </svg>
+
+          <InfoCard
+            user={display.getUserToEdit()}
+            token={this.state.authToken}
+            onClick={() => {
+              display.toggler()
+              this.forceUpdate()
+            }}
+          />
+        </Modal>
+
+        <Modal isOpen={display.getToggleDelete()}>
+          <ModalHeader>ARE YOU SURE?</ModalHeader>
+          <ModalBody>
+            Are you sure you want to delete this user? They're kinda cute and
+            they'll be gone forever if you do.
+          </ModalBody>
+          <ModalFooter>
+            <Button
+              color='danger'
+              onClick={() => {
+                this.deleteUser(display.getUserToDelete())
+                display.togglerDelete()
+                this.forceUpdate()
+              }}
+            >
+              Yes, they're dead to me
+            </Button>
+            <Button
+              color='light'
+              onClick={() => {
+                display.togglerDelete()
+                this.forceUpdate()
+              }}
+            >
+              No, they're chill
+            </Button>
+          </ModalFooter>
+        </Modal>
+      </div>
+    )
+  }
+}
+
+export default Login
```

Comment:
as mentioned before this component shouldn't be named Login. Also rename the filename from App.js. see #170